pimd/pim_rpf.c

```diff
@@ -405,17 +405,13 @@ static struct in_addr pim_rpf_find_rpf_addr(struct pim_upstream *up)
 	return rpf_addr;
 }
 
-int pim_rpf_addr_is_inaddr_none(struct pim_rpf *rpf)
+void pim_rpf_addr_any(struct pim_rpf *rpf)
 {
```

Comment:
I don't think we need this function, it is only used twice, we should just replace it with `pim_addr_to_prefix(&rpf->rpf_addr, PIMADDR_ANY);`

Reply:
(Miscounted… 3 times… but I still think it's simpler to just remove it)

Reply:
Taken care